src/Resources/skeleton/resetPassword/ResetPasswordController.tpl.php

```diff
@@ -0,0 +1,166 @@
+<?= "<?php\n" ?>
+
+namespace <?= $namespace ?>;
+
+use <?= $user_full_class_name ?>;
+use <?= $request_form_type_full_class_name ?>;
+use <?= $reset_form_type_full_class_name ?>;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use SymfonyCasts\Bundle\ResetPassword\Controller\ResetPasswordControllerTrait;
+use SymfonyCasts\Bundle\ResetPassword\Exception\ResetPasswordExceptionInterface;
+use SymfonyCasts\Bundle\ResetPassword\ResetPasswordHelperInterface;
+
+/**
+ * @Route("/reset-password")
+ */
+class <?= $class_name ?> extends AbstractController
+{
+    use ResetPasswordControllerTrait;
+
+    private $resetPasswordHelper;
+
+    public function __construct(ResetPasswordHelperInterface $resetPasswordHelper)
+    {
+        $this->resetPasswordHelper = $resetPasswordHelper;
+    }
+
+    /**
+     * @Route(name="app_forgot_password_request")
```

Comment:
It just occurred to me that it might be useful to have a short description above each method:

> Display & process form to request a password reset.

Reply:
done